### PR TITLE
footer-changes

### PR DESCRIPTION
**Pull request by acev0010**
## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
Update the footer as per the email from principle publisher

## Additional information (optional) / Information additionnelle (optionel)

**General checklist / Liste de contrôle générale**
Make your own list for the purpose of your Pull request. /// Faites votre propre liste en fonction des besoins de votre demande « pull ».

- [x] Create/update documentation /// Créer/mettre à jour la documentation
- [x] Waiting for French translation
- [ ] Waiting for review

**Related issues / Requêtes associées**
Relates to #313 

**Screenshots / Captures d'écrans**

### English version
image

### French version

image

### CODE REVIEW

**Comment by shawnthompson** [comment]
This looks awesome, thanks @acev0010. 

@emmatseng-ux, can we leave the Digital Accessibility Toolkit section out until we create the pages we need for it? If not we will have to create these pages in this pull request also.

File: src/_includes/partials/footer.njk [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
-				<li><a href="{{ footer[locale].publicServiceAndMilitaryURL }}">{{ footer[locale].publicServiceAndMilitary }}</a></li>
-				<li><a href="{{ footer[locale].newsURL }}">{{ footer[locale].news }}</a></li>
-				<li><a href="{{ footer[locale].treatiesLawsAndRegulationsURL }}">{{ footer[locale].treatiesLawsAndRegulations }}</a></li>
-				<li><a href="{{ footer[locale].governmentWideReportingURL }}">{{ footer[locale].governmentWideReporting }}</a></li>
-				<li><a href="{{ footer[locale].primeMinisterURL }}">{{ footer[locale].primeMinister }}</a></li>
-				<li><a href="{{ footer[locale].howGovernmentWorksURL }}">{{ footer[locale].howGovernmentWorks }}</a></li>
-				<li><a href="{{ footer[locale].openGovernmentURL }}">{{ footer[locale].openGovernment }}</a></li>
-			</ul>
-		</nav>
+	<h2 class="wb-inv">{{ footer[locale].aboutThisSite }}</h2>
```
Comment (shawnthompson) [inline src/_includes/partials/footer.njk:3]: The `aboutThisSite` value was removed in [src/_data/footer.js](https://github.com/gc-da11yn/gc-da11yn.github.io/pull/324/files#diff-b4613a2f8000afa0de5634f1e6a1d79a3f7590d5a474c04fdb6e896284db05da), leaving an empty `h2`.

File: src/_includes/partials/footer.njk [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
+	<div class="gc-contextual">
+	<div class="container">
+			<nav>
+				<h3>{{ footer[locale].contextualHeader }}</h3>
+				<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3"><li><a href="http://canada.ca/en">{{ footer[locale].contextualLinkText1 }}</a></li><li><a href="{{ footer[locale].contextualLink2 }}"> {{ footer[locale].contextualLinkText2 }} </a></li><li><a href="http://canada.ca/en">{{ footer[locale].contextualLinkText3 }}</a></li></ul>
+			</nav>
+		</div>
 	</div>
```
Comment (shawnthompson) [inline src/_includes/partials/footer.njk:11]: Remove contextual (DAT section) until pages have been created.

**Review by shawnthompson — CHANGES_REQUESTED** [review]
Please make the changes, thanks!

**Comment by emmatseng-ux** [comment]
Welcome back @shawnthompson ! Regarding the Contextual Digital Accessibility Toolkit, that will be a call for you and @JulieBB23 to make. The new footer was introduced [based on the research](https://blog.canada.ca/2022/12/21/wayfinding-research-project#:~:text=Government%20of%20Canada-,3.%20footer%20updates,-Available%20to%20use) the Canada.ca design system team published to support navigation as a last resort/rescue option. We determined the contents of this footer based on the treejack test that we conducted with the AWG and various CSPS employees.  

#326 and #313 - wanted to highlight the changes to the GitHub footer, thanks.

File: src/_includes/partials/footer.njk (hunk 1 above)
Comment (acev0010) [reply src/_includes/partials/footer.njk:3]: Good catch, thanks! I added the changes in the latest commit

File: src/_includes/partials/footer.njk (hunk 2 above)
Comment (acev0010) [reply src/_includes/partials/footer.njk]: Added the changes in the latest commit.

**Comment by shawnthompson** [comment]
The [Wayfinding research project improves our approach to navigation on Canada.ca | Canada.ca blog](https://blog.canada.ca/2022/12/21/wayfinding-research-project#) states the following

> The contextual band is optional and configurable. 

So I think we should leave it out for now and just focus on the mandatory Government of Canada footer in this pull request.

image

**Review by shawnthompson — APPROVED** [review]
Thanks @acev0010.

**Comment by emmatseng-ux** [comment]
@shawnthompson yup! the contextual footer is optional so it does not need to be the focus for now. However, will those pages in the footer still be developed? Those 3 include: Contact the DAT, Accessibility in Govt of Can, Accessibility in your role. 

That's a question for you and @JulieBB23 and perhaps the greater group of if these pages still align with the vision, importance that it serves to site visitors. If it's not being released now, when? Is further testing needed? - etc.

**Comment by shawnthompson** [comment]
Issue created for DAT footer section: 

[[CHANGE] Digital Accessibility Toolkit contextual footer · Issue #328 · gc-da11yn/gc-da11yn.github.io](https://github.com/gc-da11yn/gc-da11yn.github.io/issues/328)

**Comment by emmatseng-ux** [comment]
Awesome, thanks @shawnthompson !

**Pull request merged**